lib/services/saml.go

```diff
@@ -212,7 +212,7 @@ func GetSAMLServiceProvider(sc types.SAMLConnector, clock clockwork.Clock) (*sam
 	}
 
 	// adfs specific settings
-	if sc.GetProvider() == teleport.ADFS {
+	if sc.GetProvider() == teleport.ADFS || sc.GetProvider() == teleport.JumpCloud {
```

Comment:
- these are no longer 'adfs specific settings' so can we remove the comment?
- I think a switch would read better here so you don't need to call `sc.GetProvider()` twice:

```
switch sc.GetProvider() {
case teleport.ADFS, teleport.JumpCloud:
        // etc
}
```

Reply:
Fixed.